ttnn/cpp/ttnn/tensor/shape/shape_base.hpp

```diff
@@ -24,6 +24,7 @@ class ShapeBase {
     explicit ShapeBase(const std::array<uint32_t, N>& arr) : value_(arr.begin(), arr.end()) {
         init();
     }
+    explicit ShapeBase(std::span<const uint32_t> span) : value_(span.begin(), span.end()) { init(); }
```

Comment:
https://github.com/tenstorrent/tt-metal/blob/main/tests/ttnn/unit_tests/gtests/tensor/test_shape_base.cpp

Reply:
What would you like me to do here?